src/app/app.component.spec.ts

```diff
@@ -16,16 +16,16 @@ describe('AppComponent', () => {
     expect(app).toBeTruthy();
   });
 
-  it(`should have as title 'web-app'`, () => {
-    const fixture = TestBed.createComponent(AppComponent);
-    const app = fixture.componentInstance;
-    expect(app.title).toEqual('web-app');
-  });
+  // it(`should have as title 'web-app'`, () => {
```

Comment:
Once the app develops more we can modify these tests so we have a good amount of test coverage.